### PR TITLE
docs(common): add new faq entry to NgOptimizedImage docs

### PR DESCRIPTION
**Pull request by atcastle**
Small update to the NgOptimizedImage FAQ to make our stance about adding additional built-in loaders to the Angular repo more explicit. 

CC: @AndrewKushnir @kara

### CODE REVIEW

File: adev/src/content/guide/image-optimization.md [hunk 1]
```diff
@@ -400,3 +400,7 @@ The `ngSrc` attribute was chosen as the trigger for NgOptimizedImage due to tech
 The [image loaders](#configuring-an-image-loader-for-ngoptimizedimage) provider pattern is designed to be as simple as possible for the common use case of having only a single image CDN used within a component. However, it's still very possible to manage multiple image CDNs using a single provider.
 
 To do this, we recommend writing a [custom image loader](#custom-loaders) which uses the [`loaderParams` property](#the-loaderparams-property) to pass a flag that specifies which image CDN should be used, and then invokes the appropriate loader based on that flag.
+
+### Can you add a new built-in loader for my preferred CDN?
+
+We don't currently plan to support additional built-in loaders in the Angular repository. Instead, we encourage developers to publish any additional image loaders as third-party packages.
```
Comment (kara) [inline adev/src/content/guide/image-optimization.md:406]:
```suggestion
For maintenance reasons, we don't currently plan to support additional built-in loaders in the Angular repository. Instead, we encourage developers to publish any additional image loaders as third-party packages.
```

**Review by kara — APPROVED** [review]
LGTM

**Review by AndrewKushnir — APPROVED** [review]

File: adev/src/content/guide/image-optimization.md (hunk 1 above)
Comment (atcastle) [reply adev/src/content/guide/image-optimization.md]: Done!

**Review by kara — APPROVED** [review]

**Review by AndrewKushnir — APPROVED** [review]

**Comment by alxhub** [comment]
This PR was merged into the repository by commit a14d139a24a6480d4e48746ce8e4f88f0560372d.

**Pull request closed without merging**